glide.yaml

```diff
@@ -98,3 +98,4 @@ import:
   version: release-10.0
 - package: kmodules.xyz/webhook-runtime
   version: release-10.0
+- package: gomodules.xyz/envsubst
```

Comment:
You need to add `version: master`. Otherwise glide picks a random version based on dependency resolution.